nautobot/core/api/views.py

```diff
@@ -491,6 +491,9 @@ def get(self, request, *args, **kwargs):
         response = super().get(request, *args, **kwargs)
         response.data["swagger_settings"] = response.data["settings"]
         del response.data["settings"]
+
+        # Add additional data so drf-spectacular will use the Token keyword in authorization header.
+        response.data["schema_auth_names"] = ["tokenAuth"]
```

Comment:
It would be more correct to populate this into drf-spectacular (likely by implementing an [OpenApiAuthenticationExtension class](https://drf-spectacular.readthedocs.io/en/latest/customization.html#specify-authentication-with-openapiauthenticationextension)) rather than hard-coding it to this view.

(From a quick read though, this *should* be working automatically already, given that [drf-spectacular's built in `TokenScheme` says `match_subclasses = True`](https://github.com/tfranzel/drf-spectacular/blob/master/drf_spectacular/authentication.py#L35) and our `TokenAuthentication` class is a subclass of `rest_framework.authentication.TokenAuthentication`... not sure what I'm missing here)

Reply:
_Maybe_..... IF a deployment just has publicly installable plugins, I see no issue. BUT if someone installs a private package they _may_ not want to expose the schema.

I will dig more into this.